services/Cargo.toml

```diff
@@ -75,6 +75,7 @@ rand = "0.8.4"
 tempfile = "3.1"
 walkdir = "2.3"
 xml-rs = "0.8.3"
+wiremock-grpc = "0.0.3-alpha1"
```

Comment:
What is the advantage of using `wiremock-grpc` over our current mock web server?

Reply:
The regular httptest does not implement the gRPC protocol and serialization.